packages/theme/src/components/ThemeProvider.tsx

```diff
@@ -89,16 +89,18 @@ const ThemeProvider = ({
   );
 
   React.useEffect(() => {
-    const hydrogen = document.querySelectorAll(themeSelector || "[data-h2]");
+    const hydrogen = document.querySelectorAll(
+      themeSelector || "body[data-h2]",
```

Comment:
This needs to be `html` to work with the app since that is where we have the root `data-h2` attribute. I think this change was made for storybook, correct? Maybe we could try targeting both? 🤔 

```suggestion
      themeSelector || "body[data-h2], html[data-h2]",
```

Reply:
Gave this a test and it looks like it works to me. Wanna pull it down and give it a look for me?

Reply:
Works for me.  :+1: